tests/Fixtures/DoctrineType/ProductIdType.php

```diff
@@ -0,0 +1,88 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\DoctrineType;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\Type;
+use Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Entity\ProductId;
+
+final class ProductIdType extends Type
+{
+    const NAME = 'ProductId';
+
+    /**
+     * @param array            $fieldDeclaration
```

Comment:
Please, remove docblocks if they don't add any value.

Reply:
I prefer to use the extended docblocks. But for you, I deleted it. Now all is normal?